control-plane/roles/monitoring/README.md

```diff
@@ -0,0 +1,37 @@
+# monitoring
+
+This role is designed to set up monitoring for a Linux-based system using Ansible. 
```

Comment:
The role does not care about the OS, it deploys it against Kubernetes which abstracts OSs.

Reply:
fixed